libs/template/create-oryx/package.json

```diff
@@ -1,13 +1,14 @@
 {
   "name": "create-oryx",
-  "version": "0.21.0",
+  "version": "0.33.0",
   "type": "module",
   "author": "Spryker Systems GmbH",
   "license": "See license in LICENSE",
   "description": "Create Spryker Oryx Apps",
   "bin": "./bin/main.cjs",
   "devDependencies": {
-    "@spryker-oryx/cli": "*"
+    "@spryker-oryx/cli": "*",
```

Comment:
Why not `latest`?

Reply:
It doesn't make a big difference, as we build it internally anyway. But techically "*" is more correct, as we don't use latest stable release, but what we have in the monorepo at the time of the build.